### ISSUE TITLE
Conflicting types

### ISSUE THREAD

**Issue by aricus** [samdze/playdate-nim#34]
I've installed it on windows according to all the docs and I get this error no matter what I do.

playdate\api.nim:53:28: error: conflicting types for 'NimMain'; have 'void(void)'
   53 |         return -1

This is just using the test setup in the documentation

**Comment by ninovanhooff** [comment]
When I run `nimble simulator` on Windows 10, the nim compilation seems to complete fine, and a pdex**.dll** is created in the source folder. pdc complains it expects a pdex**.bin** though

**Comment by ninovanhooff** [comment]
Omg... when I add an empty `pdex.bin`  pdc stops complaining and the pdx can actually be opened on simulator without problem... wow

But that does mean that this project should properly build on Windows 10 64 bit, @aricus

I used this download to install nim: https://nim-lang.org/download/nim-1.6.16_x64.zip

After unzipping that somewhere, I ran the `finish.exe` inside of it to add nim and nimble to the Windows PATH.

I then restarted vscode and chose the `Build Sim. & Debug PDX` configuration. 
because pdc complained about not finding a pdex.bin, I added that as a dummy file; and the next time I ran the `Build Sim. & Debug PDX` it worked, and the pdx opened in simulator.

Note that I already set the environment variable for the playdate sdk
